libpangolin/include/pangolin/BfsMining/engine.h

```diff
@@ -1,10 +1,26 @@
 #include "galois/Galois.h"
 #include "pangolin/res_man.h"
 #include "pangolin/BfsMining/embedding_list.h"
+#include "llvm/Support/CommandLine.h"
+
+//! Flag that forces user to be aware that they should be passing in a
+//! symmetric graph.
+static cll::opt<bool> symmetricGraph(
+    "symmetricGraph",
+    cll::desc("Flag should be used to make user aware they should be passing a "
```

Comment:
This help message does not make sense. The flag is not for the user. It is for the system.

Reply:
Replace all such messages with either "The input graph is symmetric" or "Specify that the input graph is symmetric". 

On a different note, all help message should be consistent but they don't seem to be.

Reply:
Noted! I will modify the command option to "Specify that the input graph is symmetric."

About the help message, yes I found the several inconsistent option descriptions, as you pointed out. Do you want me to work on that through this pull request? In my opinion, separating that would be more convenient for me, because they are not related to the symmetric graph option and this pull request already consists of a lot of modifications.

Reply:
Yes, that should be a separate pull request.

Let this pull request be about graph options.